XMonad/Hooks/StatusBar.hs

```diff
@@ -390,10 +390,10 @@ statusBarPipe cmd xpp  = do
 -- >
 -- > main = xmonad $ dynamicSBs barSpawner (def { ... })
 --
--- Beware of one caveat: if you want to launch multiple status bars but use the
--- same command, you need to differentiate between them (for example, by adding
--- extra white spaces at the end). The string used to launch the commands is
--- also used to keep track of the status bars.
+-- Make sure you specify which screen to place the the status bar on (in xmobar,
```

Comment:
Undouble the "the", otherwise this is really good. Note that there's a similar comment somewhere down that should be adjusted too.

Reply:
I think I'm gonna just remove the comment there